docker/docker-compose.yml

```diff
@@ -59,7 +59,7 @@ services:
     unreal_sim:
         build:
             context: ../
-            dockerfile: ./src/unreal_sim/Dockerfile
+            dockerfile: ./src/unreal_sim/Dockerfile.sim
```

Comment:
wouldnt bother with the `.sim` here, since its already under the `unreal_sim` package - the `.something` is used in the docker folder just to differentiate all the dockefiles